src/Graph.ts

```diff
@@ -147,7 +147,7 @@ export default class Graph {
 		this.resolveId = first(
 			[
 				((id: string, parentId: string) =>
-					this.isExternal(id, parentId, false) ? false : null) as ResolveIdHook
+					parentId != null && this.isExternal(id, parentId, false) ? false : null) as ResolveIdHook
```

Comment:
is the loose `null` check intentional?

Reply:
I usually do not distinct `null` and `undefined` and even prefer to not use `undefined` value explicitly. Same true for rollup hooks. `resolveId`, `load`, etc may return both `null` and `undefined`.